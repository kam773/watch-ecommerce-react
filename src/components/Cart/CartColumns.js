import React from "react";
import { Container, Row, Col } from "reactstrap";

export default function CartColumns() {
  return (
    <Container className="container-fluid text-center d-none d-lg-block">
      <Row>
        <Col xs="10" className="mx-auto col-lg-2">
          <p className="text-uppercase font-weight-bold">products</p>
        </Col>
        <Col xs="10" className="mx-auto col-lg-2">
          <p className="text-uppercase font-weight-bold">name of product</p>
        </Col>
        <Col xs="10" className="mx-auto col-lg-2">
          <p className="text-uppercase font-weight-bold">price</p>
        </Col>
        <Col xs="10" className="mx-auto col-lg-2">
          <p className="text-uppercase font-weight-bold">quantity</p>
        </Col>{" "}
        <Col xs="10" className="mx-auto col-lg-2">
          <p className="text-uppercase font-weight-bold">remove</p>
        </Col>
        <Col xs="10" className="mx-auto col-lg-2">
          <p className="text-uppercase font-weight-bold">total</p>
        </Col>
      </Row>
    </Container>
  );
}
