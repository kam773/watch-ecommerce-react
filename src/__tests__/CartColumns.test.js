import React from 'react';
import CartColumns from '../components/Cart/CartColumns';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';

describe("CartColumnst", () => {
    it('renders without crashing', () => {
        const tree = renderer.create(<CartColumns />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('has div with class row', () => {
        const wrapper = shallow(<CartColumns/>)
        expect(wrapper.find('.row').every('.row')).toEqual(true);
    });
    it('has p tag', () => {
        const wrapper = shallow(<CartColumns/>)
        expect(wrapper.find('p').every('p')).toEqual(true);
    });
})