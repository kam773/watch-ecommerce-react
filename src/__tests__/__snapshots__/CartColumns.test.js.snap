// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartColumnst renders without crashing 1`] = `
<div
  className="container-fluid text-center d-none d-lg-block"
>
  <div
    className="row"
  >
    <div
      className="col-10 mx-auto col-lg-2"
    >
      <p
        className="text-uppercase font-weight-bold"
      >
        products
      </p>
    </div>
    <div
      className="col-10 mx-auto col-lg-2"
    >
      <p
        className="text-uppercase font-weight-bold"
      >
        name of product
      </p>
    </div>
    <div
      className="col-10 mx-auto col-lg-2"
    >
      <p
        className="text-uppercase font-weight-bold"
      >
        price
      </p>
    </div>
    <div
      className="col-10 mx-auto col-lg-2"
    >
      <p
        className="text-uppercase font-weight-bold"
      >
        quantity
      </p>
    </div>
     
    <div
      className="col-10 mx-auto col-lg-2"
    >
      <p
        className="text-uppercase font-weight-bold"
      >
        remove
      </p>
    </div>
    <div
      className="col-10 mx-auto col-lg-2"
    >
      <p
        className="text-uppercase font-weight-bold"
      >
        total
      </p>
    </div>
  </div>
</div>
`;
